src/app/core/services/mocks/mock.github.service.ts

```diff
@@ -98,7 +98,7 @@ export class MockGithubService {
    * Fabricates session data in accordance with SessionData Requirements.
    * @return Observable<{}> representing session information.
```

Comment:
Maybe can update this too since the return has been changed to `Observable<SessionData>`